src/client/styles/components/_layout.scss

```diff
@@ -19,11 +20,141 @@ fieldset {
 	padding-bottom: 10px;
 }
 
+//desktop pad-top 30 but 4 is from header leine height
 .page-section {
-	padding: 20px 0px 80px 0px;
-
+	@include media(mobile){
+		padding: 18px 5px 80px 0px;		//top: 18 + 2(header line ht)=20
+	}
 	@include media(desktop){
-		padding: 30px 0px 80px 0px;
+		padding: 26px 5px 80px 0px;   		//top :26+4(header line ht)=30
 	}
     
+}
+
+// left margin for back link in mobile
+.button--back{
+	@include media(mobile){
+	margin-left:14px;
+	
+	}
+	
+}
+
+// .page-section class from frontend-uk over written in order to do spacing changes
+.page-section * + *{
+	margin-top:0px;
```

Comment:
It prevents components having to handle spacing themselves and brings consistency to elements stacking on top of each other by only applying a top margin if there are preceding siblings.